kolibri/core/tasks/api.py

```diff
@@ -835,6 +839,20 @@ def channeldiffstats(self, request):
 class FacilityTasksViewSet(BaseViewSet):
     queues = [facility_queue]
 
+    def _job_to_response(self, job):
+        response = super(FacilityTasksViewSet, self)._job_to_response(job)
```

Comment:
Ah, yes - you do want it to change in the frontend. I'd recommend reverting these changes in job serialization, and we should make the change in cancellable on the job get saved to the task db.

Reply:
Ya, so should I save the job in the task same as I would in the api here?

Reply:
Ah well, I guess it gets enqueued.

Reply:
Yeah - the reason I rolled back on just saving the cancellable status on the job is that we don't currently have any mechanism for doing that save except for changes to the extra_metadata.